src/controllers/country.js

```diff
@@ -0,0 +1,28 @@
+import Admin from "../services/AdminServices/countryService";
+import { validation } from "../validation/countryValidation";
+
+export default class AdminController {
+  static async addCountry(req, res) {
+    try {
+      const {
+        nameOfCountry, gallery, capital, population, officialLanguage, region, currency,
+      } = req.body;
+      let countryName;
+      if (nameOfCountry) {
+        const stringCountry = String(nameOfCountry);
+        countryName = stringCountry[0].toUpperCase() + stringCountry.slice(1).toLowerCase();
+      }
+      const newCountry = {
+        nameOfCountry: countryName, gallery, capital, population, officialLanguage, region, currency
+      };
+      const { error } = validation(newCountry);
+      if (error) return res.status(400).json({ status: 400, error: error.message });
+      const country = await Admin.checkCountry(countryName);
+      if (country) return res.status(409).json({ status: 409, message: "This country already exists in the database." });
+      const createdCountry = await Admin.addCountry(newCountry);
+      return res.status(201).json({ status: 201, message: "A country has been added.", data: createdCountry, });
+    } catch (error) {
+      return res.status(500).json({ status: 500, error });
```

Comment:
This should be `return res.status(500).json({ status: 500, error: "server error or something cool to indicate a deep error" });`

Reply:
> Now, in your eslintrc file, make sure it checks for jsdoc. Now I want you to add jsdoc to every function in our repo.

Alright, will work on it now.